_src/pattern-library/sass/utilities/_helpers.scss

```diff
@@ -122,7 +122,7 @@
     );
 
     border-bottom: 1px solid $transparent;
-    color: palette(primary, base);
+    color: palette(primary, dark);
```

Comment:
@talbs The gray background is slightly too dark to provide enough contrast with this blue. Some options that might work:
* increase the size of the text
* make it bold
* lighten the gray
* darken the blue